MassSpectrometry/DissassociationTypeCollection.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections.Generic;
+
+namespace MassSpectrometry
+{
+    public enum DissassociationType
+    {
+        HCD,
+        ECD,
+        ETD,
+        CID,
+        EThCD,
+        Any,
+        Custom
+    }
+
+    public class DissassociationTypeCollection
```

Comment:
that seems good. where would you put it?

Reply:
I would put it in mzLib/MassSpectrometry/Enums/DissociationType.cs but maybe Anthony would object, since maybe that file is only intended to have the Enum in it...

Reply:
on second thought you'd have to make a class there anyway. so I would just delete the enum declared here and use the one in mzLib/MassSpectrometry/Enums/DissociationType.cs. Then here in this class, make a static dictionary which is populated appropriately which you can reference from any project using MassSpectrometry

Reply:
I deleted the enum. discovered that it was redundant with something dbailey created.

Reply:
What about the custom set? How would you deal with that? That's the case where the user picks all the fragmentation ion series that they want.

Reply:
I think at that level, the client (e.g. MetaMorpheus) would create a new Value for the "Custom" dissociation type key and populate it, because there's no way to know what the user wants to search for until it's set in MetaMorpheus